components/brave_wallet_ui/components/shared/add-custom-token-form/add-nft-form.tsx

```diff
@@ -131,6 +131,7 @@ export const AddNftForm = (props: Props) => {
         coingeckoId: '',
         logo: '',
         tokenId: tokenId ? new Amount(tokenId).toHex() : '',
+        isCompressed: false, // FIXME
```

Comment:
I put it there since hardcoding as `false` would be incorrect for manually added compressed NFTs. But since then I have updated BraveWalletService::AddUserAsset to fetch this data from SimpleHash, so this field is overwritten if it is compressed. I removed the FIXME.